### ISSUE TITLE
Add option to re-enable removed (deprecated?) "(default = X)" printing

### ISSUE THREAD

**Issue by chbrown** [scallop/scallop#103]
I was digging around the repo to see if there was a way to trigger printing the default value (when specified) in the generated `--help`, only to find in 88ae5d6d00ad39d717457438e9201d21e1e808ff that there was, but it's been deprecated!

I get what you're saying by

> it will be much prettier if developer will provide that manually however he wants

but in some cases I would prefer easy-and-automatic over manual-and-pretty. It doesn't look like it'd be trivial to shim the original `++ defVal.map("(default = %s)" format _)` back into the `wrap` arguments somehow in my consuming codebase, so, can this be made an additional argument to the `ScallopConf#opt(...)` method?

Alternatively, you could add a `ScallopConf#helpFormatter(...)` setter, and a new `org.rogach.scallop.VerboseFormatter` object with the opinionated "`(default = %s)`" preset? (I like this better than cluttering opt's argument list, and I don't expect I'd need to enable/disable the default printing on an opt-to-opt basis.)

Or maybe I haven't read enough of the source and am missing some more elegant technique for customizing the auto-generated help message?

**Comment by jo-pol** [comment]
@rvanheest @janvanmansum @lindareijnhoudt @PaulBoon

We too would prefer easy-and-automatic as default behaviour. It was pretty enough for us. This change might not be breaking compilation but it does break current behaviour and 'providing manual' requires more and harder code changes to restore old behaviour than having to add verify (acd48dcf4be4cceceaa220b23482298c456fcb3d) to upgrade.

**Comment by Rogach** [comment]
I really don't like the idea of keeping this as a default (this leaks scala-specific info into help output and is way too verbose and ugly), but providing an option to turn this on for the whole config will be reasonable. I'll implement and push this change today.

**Comment by Rogach** [comment]
Here's how code looks with currently implemented fix:

``` scala
object Conf extends ScallopConf(Seq("--apples", "1")) {
  appendDefaultToDescription = true
  val apples = opt[Int](descr = "how many apples?", default = Some(42))
  verify()
}
```

If this is ok, I will release & push new version shortly.

**Comment by chbrown** [comment]
Short & sweet. Is ok by me, thanks!

**Comment by Rogach** [comment]
Released as 1.0.1, should sync to Maven Central within an hour.
